apps/crn-server/scripts/export-entity.ts

```diff
@@ -54,8 +55,8 @@ export const exportEntity = async (
 
   do {
     records = await controller.fetch({
-      take: 50,
-      skip: (page - 1) * 50,
+      take: 20,
```

Comment:
It was failing due to query complexity. I think this might happen a lot as we add more relationships with these last entities we're migrating. So I decreased to 20, but it was a random number 😛 looking at graphql playground it could be 40. Do you guys think it's ok to leave as 20?
<img width="1375" alt="Screenshot 2023-05-19 at 15 13 22" src="https://github.com/yldio/asap-hub/assets/16595804/9c23d936-dc5a-4b5b-add7-1689d82612bf">

Reply:
20 is fine, why not, just more calls to the CMS is all